functions/googleGroupsManager.test.js

```diff
@@ -0,0 +1,70 @@
+const authenticator = require('./authenticator')
+const googleGroupsManager = require('./googleGroupsManager');
+
+// gMail API constants.
+const {google} = require('googleapis');
+const DOMAIN = 'identity-sre.com';
+const CREDENTIALS_PATH = 'credentials.json';
+
+// Testing constants
+const JEST_GROUP_EMAIL = 'jest-test@identity-sre.com';
+const JEST_GROUP_DESCRIPTION = 'jest test description';
+const JEST_GROUP_NAME = 'jest test name';
+
+test('Listing of groups is defined', async done => {
+  async function callback(auth) {
+    try {
+      let groupsData = await googleGroupsManager.listGroups(auth, google, DOMAIN);
+      expect(groupsData).toBeDefined();
+      done();
+    } catch (error) {
+        done(error);
+    }
+  }
+  authenticator.checkCredentials(CREDENTIALS_PATH, callback);
+});
+
+test('Add and delete group', async done => {
```

Comment:
Adding and deleting GSuite Google Groups was not initially in the design. Do we plan to have such a feature in the app?

Reply:
No, it's only set of internal functions for group management. I needed that to testing purposes (because there is a need to create a testing group and clean up after testing) but creating listeners for that was unnecessary. I've moved add/delete functions to test file.